client/blocks/shortcode/index.js

```diff
@@ -0,0 +1,55 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { omit } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import QueryShortcode from 'components/data/query-shortcode';
+import { getShortcode } from 'state/shortcodes/selectors';
+
+/**
+ * Local dependencies
+ */
+import ShortcodeFrame from './frame';
+
+const Shortcode = props => {
+	const { siteId, className, children, filterRenderResult, shortcode } = props;
+	const classes = classNames( 'shortcode', className );
+	let filteredShortcode = {};
+	if ( shortcode ) {
+		shortcode.body = shortcode.result;
+		filteredShortcode = filterRenderResult( omit( shortcode, 'shortcode' ) );
+	}
+	return (
+		<div>
+			<QueryShortcode siteId={ siteId } shortcode={ children } />
+			<ShortcodeFrame
```

Comment:
After the Redux refactoring, is the `ShortcodeFrame` still needed as a standalone component? Everything important happens in `connect` now and the actual `Shortcode` component is a very thin wrapper around `ShortcodeFrame`.